nilearn/maskers/multi_nifti_labels_masker.py

```diff
@@ -0,0 +1,259 @@
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details
+
+    low_pass: None or float, optional
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details
+
+    high_pass: None or float, optional
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details
+
+    t_r: float, optional
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details
+
+    dtype: {dtype, "auto"}
+        Data type toward which the data should be converted. If "auto", the
+        data will be converted to int32 if dtype is discrete and float32 if it
+        is continuous.
+
+    resampling_target: {"data", "labels", None}, optional.
+        Gives which image gives the final shape/size. For example, if
+        `resampling_target` is "data", the atlas is resampled to the
+        shape of the data if needed. If it is "labels" then mask_img
+        and images provided to fit() are resampled to the shape and
+        affine of maps_img. "None" means no resampling: if shapes and
+        affines do not match, a ValueError is raised. Defaults to "data".
+
+    memory: joblib.Memory or str, optional
+        Used to cache the region extraction process.
+        By default, no caching is done. If a string is given, it is the
+        path to the caching directory.
+
+    memory_level: int, optional
+        Aggressiveness of memory caching. The higher the number, the higher
+        the number of functions that will be cached. Zero means no caching.
+
+    n_jobs: integer, optional
+        The number of CPUs to use to do the computation. -1 means
+        'all CPUs', -2 'all CPUs but one', and so on.
+
+    verbose: integer, optional
+        Indicate the level of verbosity. By default, nothing is printed
+
+    strategy: {'sum', 'mean', 'median', 'minimum', 'maximum', 'variance',
+        'standard_deviation'}, default is 'mean'
+        The name of a valid function to reduce the region with.
+        Must be one of: sum, mean, median, minimum, maximum, variance,
+        standard_deviation
+
+    reports : boolean, optional
+         If set to True, data is saved in order to produce a report.
+         Default=True.
+
+    See also
+    --------
+    nilearn.maskers.NiftiMasker
+    nilearn.maskers.NiftiLabelsMasker
+
+    """
+
+    def __init__(self, labels_img, labels=None, background_label=0,
+                 mask_img=None, smoothing_fwhm=None, standardize=False,
+                 standardize_confounds=True, high_variance_confounds=False,
+                 detrend=False, low_pass=None, high_pass=None, t_r=None,
+                 dtype=None, resampling_target="data",
+                 memory=Memory(cachedir=None, verbose=0), memory_level=1,
+                 n_jobs=1, verbose=0, strategy="mean", reports=True):
+        self.labels_img = labels_img
```

Comment:
All of this can be removed as this class is inheriting from `NiftiLabelsMasker`. Simply call the parent constructor here.